### DISCUSSION TITLE
Error: Access denied - bin/mysql < backups/magento.sql

### DISCUSSION THREAD

**Discussion by st3phani** [markshust/docker-magento#522] [Q&A]
I have an error when I try to import the DB:

```
bin/mysql < backups/magento.sql
```

> ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privile' at line 1

**Comment by rangerz** [comment]
I anwsered before
https://github.com/markshust/docker-magento/issues/437#issuecomment-811300564
https://github.com/markshust/docker-magento/issues/133#issuecomment-612703993

**Reply by st3phani** [reply]
> I anwsered before
> [#437 (comment)](https://github.com/markshust/docker-magento/issues/437#issuecomment-811300564)
> [#133 (comment)](https://github.com/markshust/docker-magento/issues/133#issuecomment-612703993)

Thanks
